scene/main/window.h

```diff
@@ -191,6 +192,25 @@ class Window : public Viewport {
 	void _notify_theme_override_changed();
 	void _invalidate_theme_cache();
 
+	struct ThemeCache {
+		Ref<StyleBox> embedded_border;
+		Ref<StyleBox> embedded_unfocused_border;
+
+		Ref<Font> title_font;
+		int title_font_size;
```

Comment:
```suggestion
		int title_font_size = 0;
```
Primitive types should be initialized.